src/reducer.js

```diff
@@ -67,7 +68,8 @@ const DEFAULT_STATE = {
         userProfile: null,
     },
     nowUtc: localNowUtc,
-    promoCode: ''
+    promoCode: '',
+    localPromoCode: ''
```

Comment:
what is the rationale of this ?
u only need to know if the code is applied or not